spec/patterns/core_spec.rb

```diff
@@ -4,7 +4,7 @@
 
 describe "SYSLOGLINE" do
 
-  let(:value)   { "Mar 16 00:01:25 evita postfix/smtpd[1713]: connect from camomile.cloud9.net[168.100.1.3]" }
+  let(:value)   { "Mar 16 00:01:25 UNKNOWN_HOSTNAME postfix/smtpd[1713]: connect from camomile.cloud9.net[168.100.1.3]" }
```

Comment:
I like this addition, but can we please keep both? like this we're sure we match names with ```_``` and without.